### PR TITLE
Fix "sz pub get" not running for all packages.

### PR DESCRIPTION
**Pull request by Jonas-Sander**

### CODE REVIEW

File: tools/sz_repo_cli/lib/src/commands/src/pub_get_command.dart [hunk 1]
```diff
@@ -55,9 +55,7 @@ class PubGetCommand extends Command {
 
     final res = taskRunner
         .runTaskForPackages(
-          packageStream: repo
-              .streamPackages()
-              .where((package) => package.hasTestDirectory),
```
Comment (nilsreichardt) [inline tools/sz_repo_cli/lib/src/commands/src/pub_get_command.dart:60]: lol

**Review by nilsreichardt — APPROVED** [review]
LGTM

File: tools/sz_repo_cli/lib/src/commands/src/pub_get_command.dart (hunk 1 above)
Comment (Jonas-Sander) [reply tools/sz_repo_cli/lib/src/commands/src/pub_get_command.dart:60]: I copied it from the test_command code I think and forgot to remove this 🤡  
I mean should've been catched by the reviewer of the PR where I added this obviously ;P

**Review by nilsreichardt — APPROVED** [review]
LGTM

(maybe it's smarter for the next time to split this PR into two so that the fix for CI doesn't need to run through the app CI)

**Comment by Jonas-Sander** [comment]
> LGTM
> 
> (maybe it's smarter for the next time to split this PR into two so that the fix for CI doesn't need to run through the app CI)

Yeah hadn't thought about that 😅

**Pull request merged**